Android/app/src/main/java/com/vigbag/android/MainActivity.kt

```diff
@@ -2,14 +2,19 @@ package com.vigbag.android
 
 import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
+import androidx.core.splashscreen.SplashScreen.Companion.installSplashScreen
 import androidx.fragment.app.Fragment
 import com.vigbag.android.databinding.ActivityMainBinding
 
 class MainActivity : AppCompatActivity() {
     lateinit var binding:ActivityMainBinding
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        Thread.sleep(200)
+        installSplashScreen()
         setContentView(R.layout.activity_main)
+        supportActionBar?.hide()
```

Comment:
Why this?

Reply:
To remove the VigyBag actionbar at the top

Reply:
This shouldn't be needed as we are already using parent theme NoActionBar. Please investigate.